locales/en/plugin__kubevirt-plugin.json

```diff
@@ -59,5 +59,17 @@
   "Console": "Console",
   "Network Interfaces": "Network Interfaces",
   "Events": "Events",
+  "Source": "Source",
+  "Size": "Size",
+  "Drive": "Drive",
+  "Interface": "Interface",
+  "Storage Class": "Storage Class",
+  "File System Type": "File System Type",
+  "Mount Point": "Mount Point",
+  "Total Bytes": "Total Bytes",
+  "Used Bytes": "Used Bytes",
+  "The following information is provided by the OpenShift Virtualization operator.": "The following information is provided by the OpenShift Virtualization operator.",
+  "File System": "File System",
```

Comment:
shouldn't these be in sentence casing?
please change "File System" to "Filesystem" and "File System Type" I think you switch to simply "Type" :)

Reply:
The old table is File system type but if I remember correctly @upalatucci had PR in the old plugin that started changing strings to this form of presentation?